lib/matplotlib/backends/backend_mixed.py

```diff
@@ -62,27 +62,16 @@ def __init__(self, figure, width, height, dpi, vector_renderer,
 
         self._bbox_inches_restore = bbox_inches_restore
 
-        self._set_current_renderer(vector_renderer)
-
-    _methods = """
-        close_group draw_image draw_markers draw_path
-        draw_path_collection draw_quad_mesh draw_tex draw_text
-        finalize flipy get_canvas_width_height get_image_magnification
-        get_texmanager get_text_width_height_descent new_gc open_group
-        option_image_nocomposite points_to_pixels strip_math
-        start_filter stop_filter draw_gouraud_triangle
-        draw_gouraud_triangles option_scale_image
-        _text2path _get_text_path_transform height width
-        """.split()
-
-    def _set_current_renderer(self, renderer):
-        self._renderer = renderer
-
-        for method in self._methods:
-            if hasattr(renderer, method):
-                setattr(self, method, getattr(renderer, method))
-        renderer.start_rasterizing = self.start_rasterizing
```

Comment:
This was stuffing methods from the proxy renderer on to the proxied renderer.  Is it a problem that we are going to lose this?